commands/src/main/java/org/wildfly/extras/creaper/commands/elytron/CreateServerSSLContext.java

```diff
@@ -0,0 +1,386 @@
+    // Truststore
+    private final String trustStoreType;
+    private final String trustStorePath;
+    private final String trustStorePassword;
+    private final String trustStoreAlias;
+    private final String trustStoreRelativeTo;
+    private final Boolean trustStoreRequired;
+    // Multiple usage
+    private final String algorithm;         // keystore manager, truststore manager
+    private final String provider;          // keystore manager, truststore manager, keystore
+    private final String providerLoader;    // keystore manager, truststore manager, keystore
+
+
+    private CreateServerSSLContext(Builder builder) {
+        super();
+        this.name = builder.name;
+        this.keyStoreType = builder.keyStoreType;
+        this.keyStorePath = builder.keyStorePath;
+        this.keyStorePassword = builder.keyStorePassword;
+        this.keyPassword = builder.keyPassword;
+        this.trustStoreType = builder.trustStoreType;
+        this.trustStorePath = builder.trustStorePath;
+        this.trustStorePassword = builder.trustStorePassword;
+        this.protocols = builder.protocols;
+        this.cipherSuiteFilter = builder.cipherSuiteFilter;
+        this.needClientAuth = builder.needClientAuth;
+        this.wantClientAuth = builder.wantClientAuth;
+        this.authenticationOptional = builder.authenticationOptional;
+        this.securityDomain = builder.securityDomain;
+        this.maximumSessionCacheSize = builder.maximumSessionCacheSize;
+        this.sessionTimeout = builder.sessionTimeout;
+        this.keyStoreAlias = builder.keyStoreAlias;
+        this.keyStoreRelativeTo = builder.keyStoreRelativeTo;
+        this.keyStoreRequired = builder.keyStoreRequired;
+        this.trustStoreAlias = builder.trustStoreAlias;
+        this.trustStoreRelativeTo = builder.trustStoreRelativeTo;
+        this.trustStoreRequired = builder.trustStoreRequired;
+        this.algorithm = builder.algorithm;
+        this.provider = builder.provider;
+        this.providerLoader = builder.providerLoader;
+    }
+
+    @Override
+    public void apply(OnlineCommandContext ctx) throws Exception {
+
+        AddKeyStore addKeyStore = new AddKeyStore.Builder(getUniqueName(KEY_STORE_NAME))
+                .type(keyStoreType)
+                .path(keyStorePath)
+                .relativeTo(keyStoreRelativeTo)
+                .required(keyStoreRequired)
+                .aliasFilter(keyStoreAlias)
+                .provider(provider)
+                .providerLoader(providerLoader)
+                .credentialReference(new CredentialRefBuilder()
+                    .clearText(keyStorePassword)
+                    .build())
+                .build();
+
+
+        AddKeyManager addKeyManager = new AddKeyManager.Builder(getUniqueName(KEY_MANAGER_NAME))
+                // TODO solve when creaper is run on different java then server
+                .algorithm(KeyManagerFactory.getDefaultAlgorithm())
```

Comment:
I would suggest to put ternary operator here to avoid calling `algorithm` method twice